pages/nl/README.md

```diff
@@ -0,0 +1,27 @@
+# Vertaling aantekeningen
+
+Het doel van dit bestand is om enkele algemene richtlijnen te definiëren voor vertalingen die door meerdere mensen worden uitgevoerd. Zoals u wellicht weet, is het vertalen van iets van de ene taal naar de andere meestal lastig en is subjectiviteit vaak aanwezig. Het wordt dan belangrijk om expliciet gemeenschappelijke vertaalafspraken voor een bepaalde taal te vermelden, zoals bijvoorbeeld de beste manier om een bepaalde uitdrukking te vertalen of welke uitdrukkingen helemaal niet vertaald mogen worden.
+
+## Richtlijnen
+
+- _you/your_ vertalen naar _u/uw_ ipv _je/jij/jou_
+- Soms is het handiger en/of klinkt het logischer om een bepaalde uitdrukking in het engels te laten
+
+## Veel voorkomende vertalingen
+
+| engels                         | nederlands                  |
+|--------------------------------|-----------------------------|
+| _declaration_                  | verklaring (of declaratie?) |
+
+## Niet-vertaalbare woorden en uitdrukkingen
```

Comment:
Very good work maintaining that list from the ground up, really happy about it. 😊